capsicum/src/right.rs

```diff
@@ -191,11 +202,20 @@ impl RightsBuilder {
     }
 
     pub fn deny(&mut self, right: Right) -> &mut RightsBuilder {
-        self.0 = (self.0 & !(right as u64)) | 0x200000000000000;
+        let result = unsafe {
+            libc::__cap_rights_clear(&mut self.0 as *mut cap_rights_t, right as u64, 0u64)
```

Comment:
We use `﻿﻿&mut self.0 as *mut cap_rights_t` a couple times in this `impl`. It would be nice to have a method for this to reduce copy paste